destination/destination_test.go

```diff
@@ -94,6 +94,54 @@ func TestAdapter_Write(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "insert only on table without key",
+			fields: fields{
+				conn: getTestPostgres(t),
+				config: config{
+					keyColumnName: "",
+				},
+			},
+			args: args{
+				ctx: context.Background(),
+				record: sdk.Record{
+					Position: sdk.Position("28357"),
+					Metadata: map[string]string{
+						"table": "unkeyed",
+					},
+					Key: sdk.StructuredData{
+						"key": "appendonly",
```

Comment:
What's `key` here? Is it the column name?

Reply:
`key` in this case is the name of the column that corresponds to the key, but the table has no key in this case. This is to test that if a key is provided in a record, it is still recorded, even if the table has no primary key set.